apps/braze/src/components/CheckboxCard.styles.ts

```diff
@@ -0,0 +1,8 @@
+import { CSSProperties } from 'react';
+import tokens from '@contentful/f36-tokens';
+
+export const checkboxCard: CSSProperties = {
+  border: `1px solid ${tokens.gray200}`,
+  borderRadius: tokens.borderRadiusSmall,
+  height: tokens.spacingXl,
+};
```

Comment:
It might be nice to switch these styles over to using `@emotion/css` since that's used in many other apps in this repo to define css for components. Not required but just a thought.

Reply:
For example like this:
https://github.com/contentful/apps/blob/39de3b3a1964827c37565f50f0e5b9ec3967ec73/apps/ai-image-generator/frontend/src/components/config/CostSection/CostSection.styles.ts